src/ui/public/index_patterns/index.d.ts

```diff
@@ -17,4 +17,4 @@
  * under the License.
  */
 
-export { IndexPattern, StaticIndexPattern } from './_index_pattern';
+export { IndexPattern, StaticIndexPattern } from 'ui/index_patterns/_index_pattern';
```

Comment:
What is the reason for these import changes? the relative imports seemed appropriate here.

Reply:
I don't know where these changes came from... It was either part of the weird merge bullshit or VS Code did it.